Mathlib/CategoryTheory/Category/Basic.lean

```diff
@@ -106,6 +107,18 @@ open Lean Meta Elab.Tactic in
   else
     throwError "The goal does not contain `sorry`"
 
+/--
+`rfl_cat` is a macro for `intros; rfl` which is attempted in `aesop_cat` before
+doing the more expensive `aesop` tactic.
+
+Note on `refine id ?_`.
+In some cases it is important that the type of the proof matches the expected type exactly.
+e.g. if the goal is `2 = 1 + 1`, the `rfl` tactic will give a proof of type `2 = 2`.
+Starting a proof with `refine id ?_` is a trick to make sure that the proof has exactly
+the expected type, in this case `2 = 1 + 1`.
+-/
```

Comment:
Can you add a note here linking to https://github.com/leanprover/lean4/pull/7428, and explaining that if that is merged we should attempt to remove this step again (i.e. remove it and see what `!bench` says)?